### PR TITLE
yet more lint cops

### PR DESCRIPTION
**Pull request by lamont-granquist**
fixes:

Lint/AmbiguousOperator
Lint/DuplicateMethods

### CODE REVIEW

**Comment by lamont-granquist** [comment]
@chef/client-core couple more easy ones

File: lib/chef/user_v1.rb [hunk 1]
```diff
@@ -93,11 +93,6 @@ def email(arg=nil)
                     arg, :kind_of => String)
     end
 
-    def password(arg=nil)
```
Comment (stevendanna) [inline lib/chef/user_v1.rb]: I think I'm being dense.  Can someone point me at the password method that this is duplicating?

Comment (thommay) [inline lib/chef/user_v1.rb]: https://github.com/chef/chef/pull/4395/files#diff-7796095f388f8a31d90966ecaea3765aL111

Comment (stevendanna) [inline lib/chef/user_v1.rb]: Dense it is then! Thanks :D

**Comment by stevendanna** [comment]
:+1:

**Comment by thommay** [comment]
:+1:

**Comment by danielsdeleo** [comment]
:+1:

**Pull request merged**